cmd/archeio/internal/e2e/e2e_test.go

```diff
@@ -92,7 +92,7 @@ func TestMain(m *testing.M) {
 // installs tool to binDir using go install
 func goInstall(t *testing.T, tool string) {
```

Comment:
this is used on line 124 to install crane@latest, we haven't had any stability issues with crane CLI, but it did start failing due to requiring a newer go version than what we're using

we should also upgrade the go version, but it's not urgent

Reply:
gotcha!